task/dev3/Operations/SimilarSurname.java

```diff
@@ -0,0 +1,44 @@
+package task.dev3.Operations;
+
+import task.dev3.User.*;
+
+import java.io.*;
+import java.util.*;
+
+/**
+ * Class do operations for search namesakes.
+ */
+public class SimilarSurname {
+  
+  /**
+   * Search namesakes and add them in list.
+   * @param userList - list contains all added users.
+   * @return List - list, contains people with similar surnames.
+   */
+  public List similarSurname(List<User> userList) {
+    List<User> usersList = new ArrayList<User>();
+    List<User> similarSurnameList = new ArrayList<User>();
```

Comment:
why it's list of user if it's similarSurnameList ? Either rename either have List<String>